### DISCUSSION TITLE
setValue() method is giving timeout for inputting more than 7000 characters

### DISCUSSION THREAD

**Discussion by NaveenNavi14** [webdriverio/webdriverio#8286] [Q&A]
Unable to use this method (setValue()) for larger length like 7000 characters but its working fine for smaller length.

Error in the console as follows:

ERROR webdriver: Request failed with status 404 due to Error: Session not started or terminated

Failed to capture screenshot on failure.
Error: Session not started or terminated

**Comment by christian-bromann** [comment]
This is unlikely related to `setValue`. I tested it for both automation protocols and both return `7999` when running the following script:

```js
const { remote } = require('webdriverio')

;(async () => {
    const browser = await remote({
        automationProtocol: 'webdriver',
        capabilities: { browserName: 'chrome' }
    })
    await browser.url('http://guinea-pig.webdriver.io/')
    await browser.$('textarea').setValue([...Array(8000)].join('1'))

    const value = await browser.$('textarea').getValue()
    console.log(value.length)

    await browser.deleteSession()
})().catch((e) => console.log(e))
```

**Reply by NaveenNavi14** [reply]
Still getting the same error for me
